### PR TITLE
Improving chronos docs

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: en/chronos.rst [hunk 1]
```diff
@@ -134,8 +134,23 @@ You can also modify parts of a date relatively::
         ->addHours(20)
         ->subMinutes(2);
 
+It is also possible to make big jumps to defined points in time::
+
+    $time = Chronos::create();
+    $time->startOfDay();
+    $time->startOfMonth();
+    $time->endOfMoth();
```
Comment (antograssiot) [inline en/chronos.rst]: @lorenzo  endOfMo**n**th()

File: en/chronos.rst [hunk 2]
```diff
@@ -203,20 +219,48 @@ objects::
...
     // Different standard formats
-    echo $date->toAtomString();
-    echo $date->toCookieString();
-    echo $date->toIso8601String();
-    echo $date->toRfc822String();
-    echo $date->toRfc850String();
-    echo $date->toRfc1036String();
-    echo $date->toRfc1123String();
-    echo $date->toRfc2822String();
+    echo $dt->toAtomString();      // 1975-12-25T14:15:16-05:00
```
Comment (antograssiot) [inline en/chronos.rst]: I don't want to bother to much with that but all example in the page so far used ``$date`` instead of ``$dt``

**Comment by antograssiot** [comment]
:+1:

File: en/chronos.rst (hunk 2 above)
Comment (lorenzo) [inline en/chronos.rst]: May bad, I will change it

Comment (lorenzo) [inline en/chronos.rst]: @antograssiot decided to use `$time` since a plain date object would not make a lot of sense there

**Pull request merged**